saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/search/changeCourtType.js

```diff
@@ -36,25 +37,34 @@ var ChangeCourtType = (function() {
 		$("input[name='" + radioName + "']:checked").trigger("click");
 	},
 	
-	/* Show fields assigned to specified court type */
+	/* Show fields assigned to specified court type.
+	 * First on click event method is assigned to all input fields specified by name {radioName}.
+	 * Method hide all containers and triggers event to show one container related to courtType.  
+	 *  
+	 * Second method is binded only to input's specified in object {fields.button}. 
+	 * This method is triggered by event {EVENT_SHOW_CONTAINER} and when event occurs,
+	 * related container is displayed.
```

Comment:
Hmm, I don't understand the comment about. What methods are you writing about? Are you describing the inner working of assignButton method?